### DISCUSSION TITLE
Per-Instance Tablist

### DISCUSSION THREAD

**Discussion by HSGamer** [Minestom/Minestom#1021] [Help] [answered]
How can I make different tablist for each instance?
I want the tablist to only show the players at the same instance as the viewer.

**Comment by HSGamer** [comment, marked as answer]
I see noone answered so I take a look and mess with the packet.
The result: https://www.youtube.com/watch?v=6Nq-VwRCJKg
Here is my code if anyone is interesting in: https://gist.github.com/HSGamer/1f624ef36ee5221222ac60c5241cef3c

**Comment by KrystilizeNevaDies** [comment]
Currently we do not have a proper tab api, once we do, we can update this discussion with it

**Reply by HSGamer** [reply]
That would be cool. But for now, I leave the code for those who need.
